package.json

```
@@ -69,7 +69,7 @@
 		"rollup-plugin-version-injector": "^1.3.3",
 		"ts-jest": "^28.0.5",
 		"typedoc": "^0.22.18",
-		"typedoc-plugin-djs-links": "^1.1.0",
+		"typedoc-plugin-djs-links": "^1.0.1",
```

```suggestion
		"typedoc-plugin-djs-links": "^1.1.0",
```
I cant commit directly so leaving this (explanation in discord)